python/cudf/cudf/core/column/string.py

```diff
@@ -2246,6 +2246,15 @@ def get_json_object(self, json_path):
         json_path: str
             The JSONPath string to be applied to each row
             of the input column
+        allow_single_quotes: bool, default False
+            If True, representing strings with single quotes is allowed. 
+            If False, strings must only be represented with double quotes.  
+        strip_quotes_from_single_strings: bool, default True
+            If True, strip the quotes from the return value of a given row if it is a string.
+            If False, values returned for a given row include quotes if they are strings.
+        missing_fields_as_nulls: bool, default False
+            If True, when an object is queried for a field it does not contain, "null" is returned.
+            If False, when an object is queried for a field it does not contain, None is returned.
```

Comment:
Docstrings need a space on both sides of the `:`. (We plan to enforce pydocstyle rules for this in the future but haven't finished implementing it yet.)
```suggestion
        json_path : str
            The JSONPath string to be applied to each row
            of the input column
        allow_single_quotes : bool, default False
            If True, representing strings with single quotes is allowed. 
            If False, strings must only be represented with double quotes.  
        strip_quotes_from_single_strings : bool, default True
            If True, strip the quotes from the return value of a given row if it is a string.
            If False, values returned for a given row include quotes if they are strings.
        missing_fields_as_nulls : bool, default False
            If True, when an object is queried for a field it does not contain, "null" is returned.
            If False, when an object is queried for a field it does not contain, None is returned.
```